### DISCUSSION TITLE
[灌水]六分街的夜色真美啊~

### DISCUSSION THREAD

**Discussion by DerekH-233** [share121/inter-knot#1739] [其他]
一堆朋友在这聚会~
一起跳舞
![VRChat_2024-07-28_15-32-39 664_1920x1080](https://github.com/user-attachments/assets/1af500f7-467b-4612-90f7-207ce8c6032a)
![VRChat_2024-07-28_14-49-21 647_1920x1080](https://github.com/user-attachments/assets/e42f4a32-c8ca-4dc6-a578-bab23e41f573)

**Comment by lllliufeng** [comment]
太好了，有机会一定去
